[2023-11-27T13:07:09.575+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbyte_example_airflow_dag.move_raw_products_file manual__2023-11-27T13:06:00.983260+00:00 [queued]>
[2023-11-27T13:07:09.582+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbyte_example_airflow_dag.move_raw_products_file manual__2023-11-27T13:06:00.983260+00:00 [queued]>
[2023-11-27T13:07:09.583+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-27T13:07:09.636+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): move_raw_products_file> on 2023-11-27 13:06:00.983260+00:00
[2023-11-27T13:07:09.645+0000] {standard_task_runner.py:57} INFO - Started process 3285 to run task
[2023-11-27T13:07:09.648+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'airbyte_example_***_dag', 'move_raw_products_file', 'manual__2023-11-27T13:06:00.983260+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmp37evcfcs']
[2023-11-27T13:07:09.649+0000] {standard_task_runner.py:85} INFO - Job 166: Subtask move_raw_products_file
[2023-11-27T13:07:09.723+0000] {task_command.py:416} INFO - Running <TaskInstance: airbyte_example_airflow_dag.move_raw_products_file manual__2023-11-27T13:06:00.983260+00:00 [running]> on host 471f0a6f2fee
[2023-11-27T13:07:09.859+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbyte_example_***_dag' AIRFLOW_CTX_TASK_ID='move_raw_products_file' AIRFLOW_CTX_EXECUTION_DATE='2023-11-27T13:06:00.983260+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-27T13:06:00.983260+00:00'
[2023-11-27T13:07:09.860+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-27T13:07:09.860+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cp /tmp/airbyte_local/json_from_faker/_airbyte_raw_products.jsonl /tmp/airbyte_local/json_from_faker/moved_raw_products.jsonl']
[2023-11-27T13:07:09.868+0000] {subprocess.py:86} INFO - Output:
[2023-11-27T13:07:09.870+0000] {subprocess.py:93} INFO - cp: cannot create regular file '/tmp/airbyte_local/json_from_faker/moved_raw_products.jsonl': Permission denied
[2023-11-27T13:07:09.871+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-11-27T13:07:09.878+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-11-27T13:07:09.880+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=airbyte_example_***_dag, task_id=move_raw_products_file, execution_date=20231127T130600, start_date=20231127T130709, end_date=20231127T130709
[2023-11-27T13:07:09.915+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 166 for task move_raw_products_file (Bash command failed. The command returned a non-zero exit code 1.; 3285)
[2023-11-27T13:07:09.940+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-27T13:07:09.963+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
